docs/ConfigureScalarDL.md

```diff
@@ -2,156 +2,84 @@
-
-```yaml
-ledger:
-  ....
-  scalarLedgerConfiguration:
-    dbContactPoints: <Cosmos DB account endpoint>
-    dbPassword: <Cosmos DB account primary master key>
-    dbStorage: cosmos
+kubectl create secret generic scalardl --from-literal db-username=<username> --from-literal db-password=<password>
 ```
+Note-:
 
-To enable Auditor service, you can follow the [Enable Auditor](#enable-auditor) section.
+For Cosmos DB, use the account name as `username` since we can't leave the `db-uername` property empty.
 
-#### Configure scalardl-audit-custom-values
+### Configure schema-loading-custom-values
 
-To deploy the Scalar DL Auditor on Cosmos DB, you need to update the [scalardl-audit-custom-values.yaml](../conf/scalardl-audit-custom-values.yaml) file.
+To create a Scalar DL schema in the database, you need to update the [schema-loading-custom-values.yaml](../conf/schema-loading-custom-values.yaml) file.
+You can refer to the [section](#kubernetes-secret-and-schema-loading-reference) to check which values are needed based on the database used.
+If any configuration property value is defined as `N/A` in the reference table for your database, you need to remove that property from the configuration file.
 
-**With Kubernetes Secrets**
+**With Kubernetes secrets**
 
 If you have created the `scalardl` Kubernetes secret, you can use the following configuration.
 
 ```yaml
-auditor:
+schemaLoading:
   existingSecret: scalardl
-  ....
-  scalarAuditorConfiguration:
-    dbContactPoints: <Cosmos DB account endpoint>
-    dbStorage: cosmos
+  
+  database: <database>
+  contactPoints: <dbContactPoints>
+  cosmosBaseResourceUnit: <cosmosBaseResourceUnit>
+  dynamoBaseResourceUnit: <dynamoBaseResourceUnit>
+  
 ```
 
 **Without Kubernetes secrets**
 
 If you have not created the `scalardl` Kubernetes secret, you can use the following configuration.
 
-```yaml
-auditor:
-  ....
-  scalarAuditorConfiguration:
-    dbContactPoints: <Cosmos DB account endpoint>
-    dbPassword: <Cosmos DB account primary master key>
-    dbStorage: cosmos
-```
-
-To enable the Auditor service, you can follow the [Enable Auditor](#enable-auditor) section.
-
-### DynamoDB
-
-#### Configure schema-loading-custom-values
-
-To create the Scalar DL schema in DynamoDB, you need to update the [schema-loading-custom-values.yaml](../conf/schema-loading-custom-values.yaml) file.
-
-**With Kubernetes secrets**
-
-If you have created the `scalardl` Kubernetes secret, you can use the following configuration.
-
 ```yaml
 schemaLoading:
-  existingSecret: scalardl
-
-  database: dynamo
-  contactPoints: <REGION>
-  dynamoBaseResourceUnit: 10
+  database: <database>
+  contactPoints: <dbContactPoints>
+  username: <username>
+  password: <password>
+  cosmosBaseResourceUnit: <cosmosBaseResourceUnit>
+  dynamoBaseResourceUnit: <dynamoBaseResourceUnit>
 ```
 
-**Without Kubernetes secrets**
+### Ledger and Auditor reference
```

Comment:
Added a table with configuration properties required for each database to deploy schema DL and Auditor helm charts.